### DISCUSSION TITLE
[ERROR] Matlab/Simulink error

### DISCUSSION THREAD

**Discussion by woojmn** [DLR-VSDC/IEEE-MVC-2023#7] [Q&A] [answered]
Hello, I would like to discuss some questions.

We have running error with matlab.
After downloading all the files through git, run FMI_Init.m to complete FMIKit installation.
After that, an running error occurs when Main.m is executed, and an error occurs that Matlab is terminated.
The error crash log is related to the MEX-file as attached.
We tried changing the version using Matlab 2021b and 2022b, but the problem was not resolved.

How can I solve that problem?

![ERROR_log](https://user-images.githubusercontent.com/95162423/209764776-5a752ecc-f857-480e-af1f-b4f13ee6bc8a.PNG)

**Comment by JBRDLR** [comment]
Hi, please sent more information on your setup. Which OS you are using, can you try it on an other pc? Which track did you choose? When in the script the crash occurs? Also some screenshots might be useful. We do test it daily and never had this error in before. Best, Jonathan

**Reply by woojmn** [reply]
hello,
I've solved that issue. The problem was caused by the path containing a language other than English.

Thank you for the quick reply and sorry.

**Reply by grisamgit** [reply]
We are experiencing the same issue.

![image](https://user-images.githubusercontent.com/67792862/216013052-a7fccef9-4879-48e8-b06a-9fa7ae4b4f14.png)

It is a fresh Matlab installation: 9.13.0.2126072 (R2022b) Update 3.

The PC is an Intel(R) Core(TM) i7-7800X CPU @ 3.50GHz   3.50 GHz, 64-bit operating system, x64-based processor.
Windows specification:
Edition	Windows 10 Pro
Version	21H2
OS build	19044.2486
Experience	Windows Feature Experience Pack 120.2212.4190.0

We tried on two different PCs, and the same error occurs.

**Reply by LubuWithFangTianHwaHalberd** [reply]
I have experienced the same issue.
![messageImage_1675392958737](https://user-images.githubusercontent.com/121564696/216503121-50688544-c61f-4270-a8b8-bf90266d0f8a.jpg)

**Comment by JBRDLR** [comment, marked as answer]
@LubuWithFangTianHwaHalberd 
please follow this issue: 
https://github.com/DLR-VSDC/IEEE-MVC-2023/discussions/19#discussioncomment-4849751
Please make sure you did all steps (especially no special characters in the file path).

Best Jonathan
